mongo/integration/client_test.go

```diff
@@ -211,7 +211,9 @@ func TestClient(t *testing.T) {
 		mt.Run("invalid host", func(mt *mtest.T) {
 			// manually create client rather than using RunOpts with ClientOptions because the testing lib will
 			// apply the correct URI.
-			invalidClientOpts := options.Client().SetServerSelectionTimeout(100 * time.Millisecond).SetHosts([]string{"invalid:123"})
+			invalidClientOpts := options.Client().
+				SetServerSelectionTimeout(100 * time.Millisecond).SetHosts([]string{"invalid:123"}).
+				SetConnectTimeout(500 * time.Millisecond).SetSocketTimeout(500 * time.Millisecond)
```

Comment:
Added `SetConnectTimeout` and `SetSocketTimeout` because this test was taking 40 seconds to do the initial heartbeat as it's connecting to an invalid host.

Reply:
Ack